package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/controllers",
-  "version": "6.2.1",
+  "version": "6.2.2",
```

Comment:
Note that we typically bump the release version in a separate commit, rather than along with our changes. The release process we use includes making a release branch with a version bump and changelog update, which gives us a place to talk about the release as a whole, ensure the version chosen is appropriate and changes are well documented, etc.

See here for more details: https://github.com/MetaMask/controllers#release--publishing

Reply:
@Gudahtt Thanks for the education while working through this PR.